apollo/client.ts

```diff
@@ -6,8 +6,9 @@ import {
 } from "@apollo/client";
 import { request } from "http";
 
+// this has the latest subgraph for now, may need to be updated later
 const umaLinkKovan = new HttpLink({
-  uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/uma-kovan",
+  uri: "https://api.thegraph.com/subgraphs/name/nicholaspai/kovan-staging",
 });
 const umaLinkMainnet = new HttpLink({
   uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/uma",
```

Comment:
Once the [mainnet subgraph](https://thegraph.com/explorer/subgraph/nicholaspai/mainnet-contracts-staging?version=pending) is done syncing, you can change this URL to `/nicholaspai/mainnet-contracts-staging`